examples/playwright/test/basic.test.ts

```diff
@@ -5,15 +5,17 @@ import { chromium } from 'playwright'
 import type { Browser, Page } from 'playwright'
 import { expect } from '@playwright/test'
 
+const PORT = 3001
```

Comment:
In addition, we could use https://www.npmjs.com/package/get-port-please to get the available port based on the env. 3000 and 3001 can still be too common sometimes.

Reply:
Thank you for your input; I agree that it's a valid point. However, for the sake of keeping the examples as simple as possible, I would suggest maintaining their simplicity. 

The issue of port overlap arises primarily when running multiple test suites simultaneously, which is barely the case for projects utilizing vitest outside this repository. _Or maybe it isn't, then please feel free to correct me if I'm judging incorrectly_